src/main/java/com/powsybl/ws/commons/computation/utils/SpecificationUtils.java

```diff
@@ -162,6 +159,34 @@ private static <X> Specification<X> appendTextFilterToSpecification(Specificatio
         return completedSpecification;
     }
 
+    private static <X> Specification<X> generateInSpecification(String column, List<String> inPossibleValues) {
```

Comment:
```suggestion
   /**
 * Generates a specification for IN clause with the given column and values.
 * Handles large value lists by chunking them to avoid StackOverflow.
 * 
 * @param column the column name to filter on
 * @param inPossibleValues the list of values for the IN clause
 * @return a specification for the IN clause
 */
private static <X> Specification<X> generateInSpecification(String column, List<String> inPossibleValues) {

```